homeassistant/components/mqtt/client.py

```diff
@@ -533,7 +533,7 @@ def _increase_socket_buffer_size(self, sock: SocketType) -> None:
             try:
                 # Some operating systems do not allow us to set the preferred
                 # buffer size. In that case we try some other size options.
-                sock.setsockopt(socket.SOL_SOCKET, socket.SO_RCVBUF, new_buffer_size)
+                sock.setsockopt(socket.SOL_SOCKET, socket.SO_RCVBUF, new_buffer_size)  # type: ignore[union-attr]
```

Comment:
Seems still to be working when executed in the debugger. Not sure about the type warnings.